charts/redis/templates/logger-redis-secret-creds.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     "helm.sh/hook": pre-install
 data: {{ if eq .Values.global.logger_redis_location "on-cluster"}}
   password: {{ randAlphaNum 32 | b64enc }}{{ else if eq .Values.global.logger_redis_location "off-cluster"}}
-  db: {{ .Values.db | b64enc }}
-  password: {{ .Values.password | b64enc }}
-  host: {{ .Values.host | b64enc }}
-  port: {{ .Values.port | b64enc }}{{ end }}
+  db: {{ .Values.logger.redis.db | b64enc }}
```

Comment:
this doesn't work as the chart is under redis directory....i think the better fix would be removing the logger from https://github.com/deis/workflow/blob/master/charts/workflow/values.yaml#L125